.github/stale.yml

```diff
@@ -0,0 +1,15 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 365
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 90
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "Type: Feature"
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as "stale" because it has not had
+  any activity for a while. It will be closed in 90 days if no further activity occurs.
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```

Comment:
Looks like you accidentally included this file.

Reply:
Shoot, it shouldn't have... I switched branches. Sorry.

Reply:
Solved and rebased.